pypeit/spectrographs/shane_kast.py

```diff
@@ -214,13 +216,21 @@ def __init__(self):
         # Uses default primary_hdrext
         self.sky_file = 'sky_kastb_600.fits'
 
-    @staticmethod
-    def default_pypeit_par():
+    def default_pypeit_par(self):
         """
         Set default parameters for Shane Kast Blue reductions.
         """
         par = ShaneKastSpectrograph.default_pypeit_par()
         par['rdx']['spectrograph'] = 'shane_kast_blue'
+
+        # 1D wavelength solution
+        par['calibrations']['wavelengths']['rms_threshold'] = 0.15  # Might be grating dependent..
+        par['calibrations']['wavelengths']['min_nsig'] = 5.
```

Comment:
If these are the default values, then you can remove this and other lines 
that are also the defaults (and in other `spectrographs`)